modules/service/internal/types/expected_keepers.go

```diff
@@ -0,0 +1,30 @@
+package types
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/supply/exported"
+	"github.com/irisnet/irishub/modules/guardian"
+)
+
+// BankKeeper defines the expected bank keeper (noalias)
+type BankKeeper interface {
+	SendCoins(ctx sdk.Context, fromAddr sdk.AccAddress, toAddr sdk.AccAddress, amt sdk.Coins) sdk.Error
+}
+
+// GuardianKeeper defines the expected guardian keeper (noalias)
+type GuardianKeeper interface {
+	GetProfiler(ctx sdk.Context, addr sdk.AccAddress) (guardian guardian.Guardian, found bool)
+	GetTrustee(ctx sdk.Context, addr sdk.AccAddress) (guardian guardian.Guardian, found bool)
```

Comment:
should use `exported. Guardian `

Reply:
I think too. But this need be done in guardian first.